### PR TITLE
ToC Implementation - Examples chapter

### PR DESCRIPTION
**Pull request by MelwynPaul**
Reorganized the Examples chapter

### CODE REVIEW

File: content/en/docs/applications/traits/logging/logging.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
----
+docs/samples/wls-coh/todo-list---
```
Comment (gilbode) [inline content/en/docs/applications/traits/logging/logging.md:1]: We are using the terms samples and examples inconsistently.  In the Github repositories we chose to use the word examples.  We should use one or the other consistently.

File: content/en/docs/setup/platforms/OCI/OCI.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ draft: false
 
 * Create the [OKE](https://docs.cloud.oracle.com/en-us/iaas/Content/ContEng/Concepts/contengoverview.htm) cluster using the Oracle Cloud Infrastructure Console or by some other means.  
 
-* For `SHAPE`, an OKE cluster with three nodes of `VM.Standard2.4` Oracle Cloud Infrastructure [compute instance shape](https://www.oracle.com/cloud/compute/virtual-machines.html) has proven sufficient to install Verrazzano and deploy the [Bob's Books]({{< relref "/docs/samples/bobs-books.md" >}}) example application.
+* For `SHAPE`, an OKE cluster with three nodes of `VM.Standard2.4` Oracle Cloud Infrastructure [compute instance shape](https://www.oracle.com/cloud/compute/virtual-machines.html) has proven sufficient to install Verrazzano and deploy the [Bob's Books]({{< relref "docs/samples/wls-coh/bobs-books.md" >}}) example application.
```
Comment (gilbode) [inline content/en/docs/setup/platforms/OCI/OCI.md:13]: The VM.Standard2 shapes are obsolete.  We should just specify the CPU/memory/architecture requirements and let users choose themselves.  Most will likely use flex shapes.

File: content/en/docs/setup/platforms/OCI/OCI.md [hunk 3]
```diff
@@ -10,7 +10,7 @@ draft: false
 
 * Create the [OKE](https://docs.cloud.oracle.com/en-us/iaas/Content/ContEng/Concepts/contengoverview.htm) cluster using the Oracle Cloud Infrastructure Console or by some other means.
```
Comment (gilbode) [inline content/en/docs/setup/platforms/OCI/OCI.md:11]: We should probably use the full name (Oracle Container Engine for Kubernetes) on the first mention.

File: content/en/docs/setup/platforms/OCI/OCI.md (hunk 2 above)
Comment (MelwynPaul) [reply content/en/docs/setup/platforms/OCI/OCI.md:13]: Thank you, @gilbode. 
1 - I will remove the 2nd bullet that talks about 'Shape'.
2 - By specify the CPU/memory/architecture requirements - you mean - the minimum that a user will have to select and create a cluster?

File: content/en/docs/examples/wls-coh/_index.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
+---
+title: "WebLogic and Coherence"
+description: "Example applications that demonstrate Verrazzano use case scenarios based on WebLogic, Helidon, and Coherence"
```
Comment (rosemarymarano) [inline content/en/docs/examples/wls-coh/_index.md:3]: Suggestion:
-> Use case scenarios for example applications based on WebLogic, Helidon, and Coherence

**Review by rosemarymarano — COMMENTED** [review]
Two suggestions

**Review by rosemarymarano — APPROVED** [review]

**Comment by MelwynPaul** [comment]
Closing this PR. 
After opening this PR, we worked on more updates - leading to difficult to solve conflicts.
New PR: https://github.com/verrazzano/docs/pull/1070

**Pull request closed without merging**